### PR TITLE
Add iframe/pym links and Fix authentication.

### PR DESCRIPTION
**Pull request by minyk**
Add iframe and pym links for Visualization.

Fix for HTTP basic authentication. 
- Add response check for error.
- change `val request` to `var request` and `request.auth()` to `request = request.auth()`

### CODE REVIEW

File: src/main/scala/org/viz/lightning/Lightning.scala [hunk 1]
```diff
@@ -61,20 +61,24 @@ class Lightning (var host: String) extends Plots with Three {
...
     }
 
     implicit val formats = DefaultFormats
 
     val response = request.asString
-    val json = parse(response.body)
-    (json \ "id").extract[Int]
-
+    response.body.toLowerCase match {
+      case "unauthorized" => throw new Exception("Unauthoriezed. Check username and/or password.")
```
Comment (freeman-lab) [inline src/main/scala/org/viz/lightning/Lightning.scala]: typo in `Unauthoriezed`

**Comment by freeman-lab** [comment]
@minyk thanks! Both of these changes LGTM, and great to fix the auth stuff. If you just fix that one typo I'll merge in.

**Comment by minyk** [comment]
Typo fixed! Thanks.

**Comment by freeman-lab** [comment]
Perfect, thanks! Merging in.

**Pull request merged**